perfkitbenchmarker/linux_benchmarks/netperf_pps_benchmark.py

```diff
@@ -147,43 +147,38 @@ def RunNetperfAggregate(vm, server_ips):
   """
 
   # setup remote hosts file
-  vm.RemoteCommand("cd %s && rm remote_hosts"
-                   % (netperf.NETPERF_EXAMPLE_DIR))
+  vm.RemoteCommand(f"cd {netperf.NETPERF_EXAMPLE_DIR} && rm remote_hosts")
   ip_num = 0
   for ip in server_ips:
-    vm.RemoteCommand("cd %s && echo 'REMOTE_HOSTS[%d]=%s' >> remote_hosts"
-                     % (netperf.NETPERF_EXAMPLE_DIR, ip_num, ip))
+    vm.RemoteCommand(f"cd {netperf.NETPERF_EXAMPLE_DIR} && "
+                     f"echo 'REMOTE_HOSTS[{ip_num}]={ip}' >> remote_hosts")
```

Comment:
I would make a slight fix here and not do the cd and just make that part of the file:

 f"echo 'REMOTE_HOSTS[{ip_num}]={ip}' >> {netperf.NETPERF_EXAMPLE_DIR}/remote_hosts"

Hrm, this entire block could be done with vm_util.CreateRemoteFile but let's do that later.

https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/vm_util.py#L703